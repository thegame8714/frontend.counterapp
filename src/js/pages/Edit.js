import React, { Component } from "react";
import PageHeading from '../components/PageHeading';

class Edit extends Component {
	render() {
		return (
			<div className='content'>
				<PageHeading value="Edit page" />
			</div>
		);
	}
}

export default Edit;
