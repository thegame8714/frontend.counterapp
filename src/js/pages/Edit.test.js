import React from 'react';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';
import Edit from './Edit';

describe('Edit Page', () => {
  it('should render page tile as H1 with text', () => {
    const route = { pageTitle: "Test Edit page title" }
    const wrapper = mount(<Edit route={route} />);
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});