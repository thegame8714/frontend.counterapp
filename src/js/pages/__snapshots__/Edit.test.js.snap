exports[`Edit Page should render page tile as H1 with text 1`] = `
<Edit
  route={
    Object {
      "pageTitle": "Test Edit page title",
    }
  }>
  <div
    className="content">
    <PageHeading
      value="Edit page">
      <h1>
        Edit page
      </h1>
    </PageHeading>
  </div>
</Edit>
`;
